pkg/console/operator/sync_v400.go

```diff
@@ -332,10 +332,10 @@ func SyncRoute(co *consoleOperator, operatorConfig *operatorv1.Console) (*routev
 
 	// we will not proceed until the route is valid. this eliminates complexity with the
 	// configmap, secret & oauth client as they can be certain they have a host if we pass this point.
-	if len(rt.Spec.Host) == 0 {
-		// TODO STATUS
+	host := routesub.GetCanonicalHost(rt)
+	if host == "" {
 		logrus.Errorf("%q: %v \n", "route", rtErr)
-		return nil, false, errors.New("waiting on route.spec.host")
+		return nil, false, errors.New("waiting on host")
```

Comment:
I'd make this message more specific like, "waiting on admitted host for default router"